torch/onnx/symbolic_opset10.py

```diff
@@ -178,3 +178,14 @@ def flip(g, input, dims):
 
 def fmod(g, input, other):
     return g.op("Mod", input, other, fmod_i=1)
+
+@parse_args('v', 't', 'i', 'i', 'i')
+def fake_quantize_per_tensor_affine(g, inputs, scale, zero_point, quant_min=-128, quant_max=127):
+    if quant_min not in [0, -128] or quant_max not in [127, 255]:
+        _unimplemented(
```

Comment:
A minor suggestion - for code readers, it may be better to have a RuntimeError here instead of `_unimplemented` as this is something not supported in ONNX itself, and not something that is uncovered in the exporter.

Reply:
@neginraoof suggested to use `_unimplemented`.

Reply:
I agree with @neginraoof that it is better than ValueError that you had before. But _unimplemented is probably better suited for something that can be supported in the future. Since this is something not supported in ONNX (unless their is a spec change, which we can be considered), I feel that RuntimeError is a bit more appropriate.

Reply:
changed to RuntimeError.